packages/flutter/test/widgets/scrollable_selection_test.dart

```diff
@@ -11,6 +11,27 @@ import 'package:flutter_test/flutter_test.dart';
 
 import 'clipboard_utils.dart';
 
+Future<void> sendKeyCombination(
```

Comment:
Maybe we should get rid of [`sendKeys` in editable_text_test](https://github.com/flutter/flutter/blob/b20b7d995b17c130dcd7456556cd69c1fde5e858/packages/flutter/test/widgets/editable_text_test.dart#L5581) and just use this.  I like the idea of just using a SingleActivator.

Reply:
I see that you added a copy of this function to another test file.  It looks like this is already public though, maybe just add it to a shared _utils.dart file?